logprep/processor/field_manager/processor.py

```diff
@@ -0,0 +1,87 @@
+"""
+FieldManager
+============
+
+The `field_manager` processor copies or moves values from multiple source fields to one
+target field.
+Additionaly it can be used to merge multiple source field values into one target field value.
+In this process, source field lists will be merged.
+
+
+Example
+^^^^^^^
+..  code-block:: yaml
+    :linenos:
+
+    - fieldmanagername:
+        type: field_manager
+        specific_rules:
+            - tests/testdata/rules/specific/
+        generic_rules:
+            - tests/testdata/rules/generic/
+"""
+from typing import List
+from logprep.abc import Processor
+from logprep.processor.field_manager.rule import FieldManagerRule
+from logprep.util.helper import get_dotted_field_value, add_field_to, add_and_overwrite
+from logprep.processor.base.exceptions import DuplicationError
+
+
+class FieldManager(Processor):
+    """A processor that copies, moves or merges source fields to one target field"""
+
+    rule_class = FieldManagerRule
+
+    def _apply_rules(self, event, rule):
+        field_values = [
+            get_dotted_field_value(event, source_field) for source_field in rule.source_fields
+        ]
+        if None in field_values:
+            missing_fields = [
+                rule.source_fields[index]
+                for value, index in enumerate(field_values)
+                if value is None
+            ]
+            error = BaseException(f"{self.name}: missing source_fields: {missing_fields}")
+            self._handle_warning_error(event, rule, error)
```

Comment:
I thought about this too and came to the conclusion, that in future we need something like `ignore_missing_fields` as config option for the rule. But as now, this could be solved with the right filter to ensure, the fields are present. But this has the downside, that it is not possible to write a "fire and forget" rule. We should wait, if users need this behavior.